src/Navigation.js

```diff
@@ -1,131 +1,52 @@
-                                        </Dropdown>
-                                        <Nav.Link as={Link} to="/request-history">Request History</Nav.Link>
-                                        <Nav.Link as={Link} to="/organization-profile" eventKey="0">My Organization</Nav.Link>
-                                    </Nav>
-                                </Accordion.Collapse>
-                            </Card>
-                        </Accordion>
-                    </Nav.Item>
-                    {!isAuth ?
-                        <>
-                            <Nav.Item>
-                                <Card>
-                                    <Card.Header>
-                                        <Accordion.Toggle as={Link} to="/register" eventKey="0">Register</Accordion.Toggle>
-                                    </Card.Header>
-                                </Card>
-                            </Nav.Item>
-                            <Nav.Item>
-                                <Card>
-                                    <Card.Header>
-                                        <Accordion.Toggle as={Link} to="/login" eventKey="0" onClick={login}>Login</Accordion.Toggle>
-                                    </Card.Header>
-                                </Card>
-                            </Nav.Item>
-                        </>
-                        :
-                        <Nav.Item>
-                            <Card>
-                                <Card.Header>
-                                    <Accordion.Toggle as={Link} to="/" eventKey="0" onClick={logout}>Logout</Accordion.Toggle>
-                                </Card.Header>
-                            </Card>
-                        </Nav.Item>
-                    }
-                    {isAuth && <Nav.Item>
-                        <Card>
-                            <Card.Header>
-                                <Accordion.Toggle as={Link} to="/" eventKey="0">Welcome back, {user}!</Accordion.Toggle>
-                            </Card.Header>
-                        </Card>
-                    </Nav.Item>}
+      <div className="topnav">
+               <Navbar collapseOnSelect className="ml-auto" expand="lg">
+                <div className="button-right">
+               <Navbar.Toggle  aria-controls='responsive-navbar-nav' />
+               <Navbar.Collapse id="responsive-navbar-nav">
+               <Nav className="ml-auto">
+                
+                      <Link to="/" className="li">
+                         <li >About Us</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li >Services</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li >Membership</li>
+                      </Link>
+                      <Link to="/" className="li">
+                          <li>FAQs</li>
```

Comment:
This can be linked to the FAQs section which was recently added to Homepage through pr #246)
<img width="915" alt="Screen Shot 2021-06-18 at 12 01 12 am" src="https://user-images.githubusercontent.com/29667122/122411703-4e5c1b00-cfc8-11eb-83b2-403941970f63.png">